### PR TITLE
Add subscription trial end changed event

### PR DESCRIPTION
**Pull request by lexerom**
https://github.com/Rebilly/docs-product/blob/master/priorities/billing/2020-11/trial-end-time.md

### CODE REVIEW

**Review by adamaltman — DISMISSED** [review]

File: openapi/webhooks/subscription-trial-end-changed.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+summary: Subscription trial end changed
+operationId: subscription-trial-end-changed
+tags:
+  - Subscriptions
+requestBody:
+  $ref: ../components/requestBodies/Webhooks/Subscription.yaml
+responses:
+  '2XX':
+    description: Return any 2XX status to indicate that the data was received successfully
```
Comment (slavcodev) [inline openapi/webhooks/subscription-trial-end-changed.yaml:9]:
```suggestion
    description: Return any 2XX status to indicate that the data was received successfully

```

Comment (lexerom) [reply openapi/webhooks/subscription-trial-end-changed.yaml:9]: Why so? Other files do not have that line.

**Review by slavcodev — APPROVED** [review]

**Comment by lexerom** [comment]
php please review
js please review

**Review by VladimirBerdnik — APPROVED** [review]

**Comment by lexerom** [comment]
php please review

**Pull request merged**